docs/source/gbd2019_models/risk_effects/zinc_deficiency/index.rst

```diff
@@ -0,0 +1,254 @@
+            gbd_round_id=6,
+            status='best',
+            decomp_step='step4')
+
+  pafs = get_draws(
+            gbd_id_type=['rei_id', 'cause_id'], 
+            gbd_id=[94, 302],
+            source='burdenator',
+            measure_id=2, #dalys
+            metric_id=2, #percent
+            year_id=2019,
+            gbd_round_id=6,
+            status='best',
+            decomp_step='step5')
+
+.. note::
+
+  In GBD 2019, the stored PAF for deaths and YLLs is slightly greater than the stored PAF for DALYs. There is no stored PAF for YLDs. The specified measure in the code snippet above is DALYs to be conservative; however, we should talk with the GBD modeler about why this is, as there do not appear to be different relative risks for morbidity and mortality in the data available. 
+
+.. list-table:: Relative Risks
+   :widths: 5 5 5
+   :header-rows: 1
+
+   * - Exposure Category
+     - Relative Risk
+     - Note
+   * - cat1 (zinc deficient)
+     - 1.14 (1.07, 1.21)
+     - Specific to 1-4 year olds, does not vary by sex 
+   * - cat2 (not zinc deficient)
+     - 1
+     - TMREL
+
+Validation and Verification Criteria
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. todo::
+
+  List validation and verification criteria, including a list of variables that will need to be tracked and reported in the Vivarium simulation to ensure that the risk outcome relationship is modeled correctly
+
+Assumptions and Limitations
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. todo::
+
+	List assumptions and limitations of this modeling strategy, including any potential issues regarding confounding, mediation, effect modification, and/or generalizability with the risk-outcome pair.
+
+Bias in the Population Attributable Fraction
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+As noted in the `Population Attributable Fraction` section of the :ref:`Modeling Risk Factors <models_risk_factors>` document, using a relative risk adjusted for confounding to compute a population attributable fraction at the population level will introduce bias.
+
+.. todo::
+
+	Outline the potential direction and magnitude of the potential PAF bias in GBD based on what is understood about the relationship of confounding between the risk and outcome pair using the framework discussed in the `Population Attributable Fraction` section of the :ref:`Modeling Risk Factors <models_risk_factors>` document.
+
+References
+----------
+
+.. todo::
+
+  Update references to GBD 2019 once published
+
+.. todo::
+
+  Update the GBD 2017 Risk Factor Methods appendix citation to be unique to your risk effects page (replace 'Risk-Effects-Model-Template' with '{Risk Name}-Effects')
+
+  Update the appropriate page numbers in the GBD risk factors methods appendix below
+
+  Add additional references as necessary 
+
+.. [GBD-2017-Risk-Factors-Appendix-Risk-Effects-Model-Template]
+
+   Pages ???-??? in `Supplementary appendix 1 to the GBD 2017 Risk Factors Capstone <risk_factors_methods_appendix_>`_:
```

Comment:
This reference needs to be updated to the 2019 risk appendix. The page numbers for zinc deficiency are 184-185.